core-java-modules/core-java-concurrency-simple/src/main/java/com/baeldung/concurrent/executorservice/CallableTask.java

```diff
@@ -0,0 +1,10 @@
+package com.baeldung.concurrent.executorservice;
+
+import java.util.concurrent.Callable;
+
+public class CallableTask implements Callable<String> {
+    @Override
+    public String call() throws Exception {
```

Comment:
where is this code snippet used in the article?

Reply:
This code snippet is used at ScheduledExecutorServiceDemo.class (section 6). I did not add the code snippet in the article because the point of that section is to showcase the methods : scheduled(), scheduleAtFixedRate() and scheduleWithFixedDelay(). Let me know if we should reference it at the article.

Reply:
Ok